doc/persistence.rst

```diff
@@ -79,6 +92,16 @@ are 'gzip', 'bz2', 'lzma' and 'xz':
 
     Lzma and Xz compression methods are only available for python versions >= 3.3.
 
+Compressor files provided by the python standard library can also be used to
+compress pickle in a context manager, e.g `gzip.GzipFile`, `bz2.BZ2File`,
+`lzma.LZMAFile`:
+    >>> # Dumping in a gzip.GzipFile file object using a compress level of 3.
```

Comment:
Done, I also used `compresslevel=3` in the line below to match what is written in the comment.